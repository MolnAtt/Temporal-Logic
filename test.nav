\beamer@endinputifotherversion {3.27pt}
\headcommand {\sectionentry {1}{HF}{1}{HF}{0}}
\headcommand {\beamer@sectionpages {1}{0}}
\headcommand {\beamer@subsectionpages {1}{0}}
\headcommand {\beamer@subsectionentry {0}{1}{1}{1}{}}\headcommand {\beamer@subsectionpages {1}{0}}
\headcommand {\sectionentry {2}{Ockhamist axioms}{2}{Ockhamist axioms}{0}}
\headcommand {\beamer@sectionpages {1}{1}}
\headcommand {\beamer@subsectionpages {1}{1}}
\headcommand {\beamer@subsectionentry {0}{2}{1}{2}{}}\headcommand {\beamer@subsectionpages {2}{1}}
\headcommand {\slideentry {2}{1}{2}{3/7}{}{0}}
\headcommand {\beamer@framepages {3}{7}}
\headcommand {\slideentry {2}{1}{3}{8/8}{}{0}}
\headcommand {\beamer@framepages {8}{8}}
\headcommand {\sectionentry {3}{Irreflexivity}{9}{Irreflexivity}{0}}
\headcommand {\beamer@sectionpages {2}{8}}
\headcommand {\beamer@subsectionpages {2}{8}}
\headcommand {\beamer@subsectionentry {0}{3}{1}{9}{}}\headcommand {\beamer@subsectionpages {9}{8}}
\headcommand {\slideentry {3}{1}{2}{10/12}{}{0}}
\headcommand {\beamer@framepages {10}{12}}
\headcommand {\slideentry {3}{1}{3}{13/20}{}{0}}
\headcommand {\beamer@framepages {13}{20}}
\headcommand {\slideentry {3}{1}{4}{21/21}{}{0}}
\headcommand {\beamer@framepages {21}{21}}
\headcommand {\slideentry {3}{1}{5}{22/22}{}{0}}
\headcommand {\beamer@framepages {22}{22}}
\headcommand {\slideentry {3}{1}{6}{23/23}{}{0}}
\headcommand {\beamer@framepages {23}{23}}
\headcommand {\slideentry {3}{1}{7}{24/24}{}{0}}
\headcommand {\beamer@framepages {24}{24}}
\headcommand {\sectionentry {4}{Irr. can. submodel}{25}{Irr. can. submodel}{0}}
\headcommand {\beamer@sectionpages {9}{24}}
\headcommand {\beamer@subsectionpages {9}{24}}
\headcommand {\beamer@subsectionentry {0}{4}{1}{25}{}}\headcommand {\beamer@subsectionpages {25}{24}}
\headcommand {\slideentry {4}{1}{2}{26/26}{}{0}}
\headcommand {\beamer@framepages {26}{26}}
\headcommand {\slideentry {4}{1}{3}{27/27}{}{0}}
\headcommand {\beamer@framepages {27}{27}}
\headcommand {\slideentry {4}{1}{4}{28/28}{}{0}}
\headcommand {\beamer@framepages {28}{28}}
\headcommand {\slideentry {4}{1}{5}{29/34}{}{0}}
\headcommand {\beamer@framepages {29}{34}}
\headcommand {\slideentry {4}{1}{6}{35/36}{}{0}}
\headcommand {\beamer@framepages {35}{36}}
\headcommand {\slideentry {4}{1}{7}{37/38}{}{0}}
\headcommand {\beamer@framepages {37}{38}}
\headcommand {\slideentry {4}{1}{8}{39/39}{}{0}}
\headcommand {\beamer@framepages {39}{39}}
\headcommand {\sectionentry {5}{Logics}{40}{Logics}{0}}
\headcommand {\beamer@sectionpages {25}{39}}
\headcommand {\beamer@subsectionpages {25}{39}}
\headcommand {\beamer@subsectionentry {0}{5}{1}{40}{}}\headcommand {\beamer@subsectionpages {40}{39}}
\headcommand {\slideentry {5}{1}{2}{41/41}{}{0}}
\headcommand {\beamer@framepages {41}{41}}
\headcommand {\slideentry {5}{1}{3}{42/42}{}{0}}
\headcommand {\beamer@framepages {42}{42}}
\headcommand {\slideentry {5}{1}{4}{43/43}{}{0}}
\headcommand {\beamer@framepages {43}{43}}
\headcommand {\beamer@partpages {1}{43}}
\headcommand {\beamer@subsectionpages {40}{43}}
\headcommand {\beamer@sectionpages {40}{43}}
\headcommand {\beamer@documentpages {43}}
\headcommand {\def \inserttotalframenumber {23}}
