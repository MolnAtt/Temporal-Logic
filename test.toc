\beamer@endinputifotherversion {3.27pt}
\beamer@sectionintoc {1}{HF}{1}{0}{1}
\beamer@sectionintoc {2}{Ockhamist axioms}{2}{0}{2}
\beamer@sectionintoc {3}{Irreflexivity}{9}{0}{3}
\beamer@sectionintoc {4}{Irr. can. submodel}{25}{0}{4}
\beamer@sectionintoc {5}{Logics}{40}{0}{5}
