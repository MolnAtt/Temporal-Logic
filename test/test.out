\BOOKMARK [2][]{Outline0.1}{HF}{}% 1
\BOOKMARK [2][]{Outline0.2}{Ockhamist axioms}{}% 2
\BOOKMARK [2][]{Outline0.3}{Irreflexivity}{}% 3
\BOOKMARK [2][]{Outline0.4}{Irr. can. submodel}{}% 4
\BOOKMARK [2][]{Outline0.5}{Logics}{}% 5
